### ISSUE TITLE
Create alerts for Testnet

### ISSUE THREAD

**Issue by mlegner** [MystenLabs/walrus#1046]
- [X] Issue higher-urgency alert if > 200 shards are unavailable. (This may depend on #1088.)
- [X] Exclude nodes that are not part of the current committee from alerts. (This may also depend on #1088.)
- [X] Low-urgency alert if nodes don't send metrics. -> this is implicitly covered by the way we set up the alerts
- [ ] Alerts for 5xx errors?
- [ ] Alerts for aggregators/publishers?
